SETUP.md

```diff
@@ -10,6 +10,11 @@ Ensure that you have docker and docker-compose installed on your machine, as wel
   - “Install now”
   - Bypass character count limit option
   - You should be able to open a Python 3.6 Desktop App and run lines of Python
+- PiP
+  - Download get-pip.py ( https://bootstrap.pypa.io/get-pip.py )
```

Comment:
I'm not sure to understand what to do here. Could you please add more details on that?

Reply:
Added additional steps in Setup:

- Go to https://bootstrap.pypa.io/
- Right click on 'get-pip.py' -> 'Save link as...'
- Select a download location